### PR TITLE
Remove routing if no appserver is running

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/1810/

**Comment by cdonati** [comment]
:+1:

File: AppController/djinn.rb [hunk 1]
```diff
@@ -983,13 +983,15 @@ def status(secret)
         next if app_name == "none"
         stats_str << "    Information for application: #{app_name}\n"
         stats_str << "        Language            : "
-        unless @app_info_map[app_name]['language'].nil?
-          stats_str << "#{@app_info_map[app_name]['language']}\n"
-        else
+        if @app_info_map[app_name]['language'].nil?
```
Comment (menivaitsi) [inline AppController/djinn.rb]: Is this the best check to ensure app isn't running? Can we instead use just @app_info_map[app_name]?

Comment (menivaitsi) [inline AppController/djinn.rb]: I understand that if the language is not set, we report stats unknown. Correct?

Comment (obino) [inline AppController/djinn.rb]: This is just reporting, and here we just report the language of the application. This is set way before the appserver is running, so technically we can see a 'pending' appserver (that is the app is not running), but we can still have the language showing properly. The case (rare) is that this call is done after the update call has been done, so the app_info_map for the app is created, but the language has not been set yet.

Comment (menivaitsi) [inline AppController/djinn.rb]: @obino Thanks for clarifying.

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/1820/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/1827/

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/1834/

**Pull request merged**